app/src/main/res/values/strings.xml

```diff
@@ -71,8 +71,8 @@
     <string name="lwp_drag">Emulate drag event for wallpaper</string>
     <string name="lwp_drag_desc">Send multiple touch events</string>
     <string name="wp_drag_animate">Scroll wallpaper</string>
-    <string name="wp_drag_animate_desc">Drag left-right will scroll wallpaper</string>
-    <string name="icons_hide_main">Hide application icons</string>
+    <string name="wp_drag_animate_desc">Drag → to scroll wallpaper</string>
```

Comment:
One can drag left and right, not sure why you chose only a right arrow

Reply:
Misinterpreted that one